### ISSUE TITLE
Multiple States: Django crash when querying district boundaries

### ISSUE THREAD

**Issue by parose** [openstates/openstates#1717]
When accessing district boundary information in the API, certain districts return a Python traceback: "Piston/0.2.3rc1 (Django 1.9.13) crash report:\n\nTraceback (most recent call last):\n\n  File \"/home/openstates/virt/local/lib/python2.7/site-packages/newrelic/hooks/component_piston.py\", line 33, in __call__..."

This issue has occurred across multiple states, including WA, FL, and TX, and only occurs for some districts. In addition, some districts that return a crash report at one time will return correct information at another.

**Comment by jamesturk** [comment]
this is a duplicate of #1570 - currently blocked by the API key rollout, we'll be throttling the responsible users in the near future
